models/staging/edfi_3/base/base_ef3__parents.sql

```diff
@@ -20,6 +20,11 @@ renamed as (
         v:maidenName::string                                                             as maiden_name,
         v:generationCodeSuffix::string                                                   as generation_code_suffix,
         v:personalTitlePrefix::string                                                    as personal_title_prefix,
+        -- the following three fields were introduced to the Contacts resource, which replaced Parents in v5.0
```

Comment:
We could, but I think it would be more awkward because union_relations requires the entities you're unioning to be tables or materialized views, not CTEs. I would need to do the rename from parent_unique_id to contact_unique_id either here (probably too breaking) or in a separate model somewhere before using union_relations. This seemed cleaner to me.